resources/js/mosaic.toolbar.js

```diff
@@ -121,7 +121,7 @@ class Toolbar {
         var self = this;
 
         var selected_tile = self.mosaic.document.querySelector(".mosaic-selected-tile");
-        var tiletype = selected_tile ? selected_tile["mosaic-tile"].getType() : null;
+        var tiletype = selected_tile ? $(selected_tile).data("mosaic-tile").getType() : null;
```

Comment:
It shouldn't make a difference if we use `tile["mosaic-tile"]` or `$tile.data("mosaic-tile")` (besides performance) ... like in patternslib both is initialized. If that makes a difference, there might be something else wrong.